src/views/accounts/profile/YourProfile.js

```diff
@@ -151,43 +224,32 @@ export const YourProfile = ({ getUser }) => {
   React.useEffect(() => {
     getUserEmails();
   }, []);
+  const headerProps = {
+    title: 'Profile',
+    description: `Manage your Profile`,
+  };
   return (
-    <Box paddingY={4}>
-      <Grid container gap={4}>
-        <Grid item>
-          <form noValidate onSubmit={formik.handleSubmit}>
-            <FormInput
-              name={'firstName'}
-              label={userInfo?.firstName}
-              formik={formik}
-            />
-            <FormInput
-              name={'lastName'}
-              label={userInfo?.lastName}
-              formik={formik}
-            />
-            <Button color="primary" variant="contained" fullWidth type="submit">
-              Submit
-            </Button>
-          </form>
-        </Grid>
-
-        <Grid item>
-          <ProfileHeader userInfo={userInfo} />
-        </Grid>
-      </Grid>
-
-      <Box>
-        <CustomTable
-          deleteEmail={deleteEmail}
-          userInfo={userInfo}
-          emails={emails}
-        />
-      </Box>
-      <Box>
+    <Grid container>
+      <AccountsHeader {...headerProps}>
         <Button
           color="primary"
           variant="contained"
+          onClick={() => {
```

Comment:
you can use the startIcon props on Edit Profile, and Add Email button on adding Icon
https://mui.com/material-ui/react-button/#IconLabelButtons.js